.travis.yml

```diff
@@ -21,7 +21,7 @@ script:
   - docker-compose run --rm webtest "rubocop -R --config .rubocop_todo.yml"
   - docker-compose run --rm webtest "rake db:drop db:create db:migrate db:seed"
   - docker-compose run --rm webtest "rails test"
-  - docker-compose run --rm webtest "rake db:drop db:create db:migrate db:reset"
+  - docker-compose run --rm webtest "rake db:drop db:create db:migrate db:seed"
```

Comment:
Do not run `db:seed`. Database should be empty for tests.

Reply:
Why is `db:seed` listed before `rails test` then? From the rails docs, `db:seed` is for loading data that should appear in prod. It is not the same as fixtures.

Reply:
Because the old tests assume existing data, which they should not.

Reply:
We should run seed nor should we have loaded data in the migrations (which is why the reset is there).

Reply:
As discussed offline, I was wrong about the seeding... `db:reset`, is intended to remove all data from the data from the database to avoid data added by migrations. 
Since the test are not failing and I do not remember a case that was failing because of this I will approve the change.